Common/SAM.h

```diff
@@ -339,15 +339,7 @@ struct SAMRecord : SAMAlignment {
 			>> o.cigar >> o.mrnm >> o.mpos >> o.isize;
 #if SAM_SEQ_QUAL
 		in >> o.seq >> o.qual;
-		std::string allTags = "";
-		if (in.peek() != '\n')
-			in >> allTags;
-		while (in.peek() != '\n' && !in.eof()) {
-		    std::string tag = "";
-			in >> tag;
-			allTags += "\t" + tag;
-		}
-		o.tags = allTags;
+		getline(in, o.tags);
```

Comment:
LGTM. Are there any places in the code that assumes that `.tags` contains only a single tag?

Reply:
As far as I can tell, the tags variable is only used to save the tags output.

Reply:
Look for other locations that extract `BX:Z:` from the SAM tags, and ensure they can handle multiple SAM tags.